### ISSUE TITLE
Docker setup pgpass file

### ISSUE THREAD

**Issue by foosinn** [OpenSlides/OpenSlides#5804]
**Describe the bug**

`server/docker/entrypoint-db-setup` seems to ignore my postgres configuration:

```
# Configure database
# TODO: env variables??
echo "postgres:5432:instancecfg:openslides:openslides" > "${HOME}/.pgpass"
chmod 600 "${HOME}/.pgpass"
```

**How to Reproduce**

Run docker setup.

**Expected behavior**

Using my Database configuration from environment.

**System information**

- OpenSlides-Version: 3.3
- Python Version: Docker Image provided

**Additional context**

I did a 1 to 1 copy of the docker deployment to kubernetes and tripped over this configuration.

**Comment by foosinn** [comment]
Link:
https://github.com/OpenSlides/OpenSlides/blob/010b61cce255102779a976e6a9780fc6d3f9b756/server/docker/entrypoint-db-setup#L27-L30

**Comment by gsiv** [comment]
I believe this code is no longer required or used but left over from when the backend used to fetch its configuration files from the database.  Since then, we have switched to an environment variable-based setup procedure.  This code should simply be removed, if I'm not mistaken (I have not tested this yet).

Whatever problem you're facing probably has nothing to do with pgpass.

When you say you expect the container to use your configuration from the environment, do you mean by that that you configured the following variables in `.env`?

```
DATABASE_HOST=
DATABASE_PASSWORD=
DATABASE_PORT=
DATABASE_USER=
```

**Comment by foosinn** [comment]
The next codeblock validates the configuration with pg_isready:

https://github.com/OpenSlides/OpenSlides/blob/010b61cce255102779a976e6a9780fc6d3f9b756/server/docker/entrypoint-db-setup#L32-L35

I replaced pg_isready with `/bin/true`, then the Image worked. If the file is depricated the pg_isready check should be handled diffrently.

**Comment by gsiv** [comment]
> If the file is deprecated the pg_isready check should be handled differently.

No credentials are required for `pg_isready`, so `.pgpass` can and should be removed.  I have verified this.

> I replaced pg_isready with /bin/true, then the Image worked.

That's good to know!  You mentioned earlier that you started the stack on Kubernetes.  Is the database, i.e., `pgbouncer`, available as `db`?  In the provided configurations `db` is an alias for `pgbouncer`.  Perhaps this detail was lost in your setup?

Should we maybe replace `db` with yet another environment variable here?

**Comment by foosinn** [comment]
I setup a service called db to provide the hostname. Still i was unable to get past pg_isready until i replaced it with /bin/true.

**Comment by foosinn** [comment]
The deployment I setup:
https://github.com/foosinn/OpenSlides/tree/k8s/docker/k8s

**Comment by gsiv** [comment]
> The deployment I setup:

Thanks for sharing!

> I setup a service called db to provide the hostname. Still i was unable to get past pg_isready until i replaced it with /bin/true.

Unfortunately, I can't tell you why your service setup doesn't work.  The check definitely works *if* `db` is available.  I would log into the container next and see if `host`, `ping`, etc. work as expected.

I have also updated the entrypoint based on what we have found in this issue, however; cf. https://github.com/OpenSlides/OpenSlides/pull/5815.  `pg_isready` now checks the configured database which might also be a solution to your problem.  In this case, you would not have to worry about the `db` alias anymore.
